qpc/source/utils.py

```diff
@@ -89,6 +87,10 @@ def build_source_payload(args, add_none=True):
             options = {'ssl_protocol': args.ssl_protocol}
         else:
             options['ssl_protocol'] = args.ssl_protocol
+    if hasattr(args, 'use_paramiko') and args.use_paramiko is not None:
+        # Don't have to to check if options is None because paramiko is the
+        # only valid option for network source
+        options = {'use_paramiko': args.use_paramiko}
```

Comment:
Let's add a None check here so that the ssl options are not removed.  The server will respond with a validation error, but that is okay.  It will tell the user what they should change.